internal/powerstore/powerstore.go

```diff
@@ -50,7 +50,7 @@ func New(ctx context.Context, ec ec.Backend, ds datastore.Datastore, cs *certsto
 	return &Store{
 		Backend: ec,
 
-		ds:         namespace.Wrap(ds, datastore.NewKey("/ohshitstore")),
+		ds:         namespace.Wrap(ds, datastore.NewKey("/powerstore")),
```

Comment:
This is the most significant change here, where it effectively clears the store.